index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
+      <a class="header--logo" href="#">
         <img src="images/keys.png" alt="keyboard" />
       </a>
-      <ul class="nav-links-cont">
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+      <nav class="header--nav">
```

Comment:
Got rid of the list (li) elements. This is a weird one because a lot of sites suggest you do this but I don't think it really makes a difference. Just seems like extra markup to me